website/docs/d/scc_posture_scopes.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_scopes"
+description: |-
+  Get information about list_scopes
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_scopes
+
+Provides a read-only data source for list_scopes. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```

Comment:
1. Review information of Security and Compliance Center posture scopes. 
2. Add reference link about posture scopes